features/rt/hack/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+assetsDir=$(dirname "$0")/../assets
+manifestsDir=$(dirname "$0")/../manifests
+
+for node in $(oc get nodes --selector='!node-role.kubernetes.io/master' -o name); do 
+  oc label $node node-role.kubernetes.io/worker-rt=""
+done
+oc create -f $manifestsDir/00-mcp-worker-rt.yaml
+oc create -f $manifestsDir/06-mc-rtkernel-worker-rt.yaml
```

Comment:
Can this be a one stop shop for all other manifests ?  (e.g. performance, cpu isolation)  if so this needs to move to an upper level.

Reply:
fixed by moving the mcp to its own feature directory